packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -148,14 +154,17 @@ class PreviewCommand extends ShorebirdCommand {
     String? deviceId,
   }) async {
     const platform = ReleasePlatform.android;
-    final aabPath = getArtifactPath(
-      appId: appId,
-      release: release,
-      platform: platform,
-      extension: 'aab',
+    final aabFile = File(
+      getArtifactPath(
+        appId: appId,
+        release: release,
+        platform: platform,
+        extension: 'aab',
```

Comment:
isn't `extension` a keyword? We might want to rename that property.

Reply:
This wasn't introduced in this PR, but seems fine aside from the syntax highlighting?